src/resources/elements/lazylinkfield.js

```diff
@@ -103,6 +108,10 @@ export class LazyLinkfield extends Field {
    */
   setValue(value) {
     if (!this.child) {
+      // Caching values helps when using setValue() in a big form. By caching
```

Comment:
This is a really great comment. I understand the what, why, and how of this code.